sys-process/dcron/dcron-4.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cron toolchain-funcs systemd
+
+DESCRIPTION="Dillon's lightweight and minimalist cron daemon"
+HOMEPAGE="
+	https://github.com/ptchinster/dcron
+	http://www.jimpryor.net/linux/dcron.html
+	https://apollo.backplane.com/FreeSrc/
+"
+SRC_URI="https://github.com/ptchinster/dcron/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
```

Comment:
`~ia64` has been dropped from Gentoo. Could you rebase and drop this keyword completely?